src/components/20-molecules/dropdown/index.scss

```diff
@@ -39,6 +39,13 @@ $dropdown-height: 40px;
 axa-dropdown {
   display: block;
 
+  &[usecase="datepicker"] {
```

Comment:
I do set min-width as default, which I want to preserve for backwards compatibility. I need to make datepicker's usage of dropdown an exception case to prevent visual irritation (non-tight focus border).